src/veloxi-plugin/SwapyPlugin.ts

```diff
@@ -506,6 +506,7 @@ export const SwapyPlugin: PluginFactory<SwapyConfig, SwapyPluginApi> = (
       hasSwapped = false
       slotItemMapOnDragStart = null
       activeScrollContainer?.endScrollTracking()
```

Comment:
Deleting the activeScrollContainer so that scroll tracking stops completely

Otherwise, the onMouseMove listener will keep calculating things in the background, leading to possible performance weirdness and maybe bugs. This keeps everything cleaned up in the drag lifecycle.